block/argocd-sync.yaml

```diff
@@ -58,4 +58,5 @@ spec:
           argocd app wait {{inputs.parameters.application-name}} --health {{inputs.parameters.flags}}
           ARGOCD_REV=$(argocd app history {{inputs.parameters.application-name}} | grep -o "(.*)" | tail -1 | sed 's/[()]//g')
           ARGOCD_APP={{inputs.parameters.application-name}}
-          echo "{\"$ARGOCD_APP\": \"$ARGOCD_REV\"}" > greenops-output.txt
+          ARGOCD_ADDR={{inputs.parameters.argocd-server-address}}
+          echo "{\"$ARGOCD_APP\": \"$ARGOCD_REV\", \"argocd-address\": \"$ARGOCD_ADDR\"}" > greenops-output.txt
```

Comment:
You can just use the `ARGOCD_SERVER` variable here instead of making a new ARGOCD_ADDR variable

Reply:
Updated